sidecar/proxy/envoy/manager.go

```diff
@@ -283,6 +246,90 @@ func (m *manager) generateConfig(rules []api.Rule, instances []api.ServiceInstan
 	}, nil
 }
 
+func BuildRDSClusters(tcpProxyList []config.TCPProxyConfig, sdsPort int) []Cluster {
```

Comment:
function should be named `buildStaticClusters` since it is building more than just the `rds` cluster and it shouldn't need to be exported since it is only called in `generateConfig()`